app/models/member.rb

```diff
@@ -72,17 +72,15 @@ def liquid_data
                                     welcome_name: full_name.blank? ? email : full_name)
   end
 
-  def publish_subscription
-    ChampaignQueue.push(
-      type: 'subscribe_member',
-      params: {
-        email: email,
-        name: name,
-        country: country,
-        postal: postal
-
-      }
-    )
+  def publish_subscription(locale = nil)
```

Comment:
ahh fuck I have to push anyway for rubocop... I'll change it to `publish_signup`